src/types/config/SidecarConfig.ts

```diff
@@ -9,7 +9,10 @@ export interface ISidecarConfig {
 
 interface ISidecarConfigSubstrate {
 	WS_URL: string;
-	CUSTOM_TYPES: Record<string, string> | undefined;
```

Comment:
I think here it would make sense to use the polkadot-js helper type `AnyJson`: https://github.com/polkadot-js/api/blob/94a4594d320753054faf7f4ec92a362cc6603d7e/packages/types/src/types/helpers.ts#L10

This essentially is a recursively defined type that should align with any valid JSON.

We could import it at the top with
```
import { AnyJson } from '@polkadot/types/types';
```

This line could then change to
```
CUSTOM_TYPES: AnyJson
```

Reply:
Thanks. This is cool, and solves the problem. Will resubmit in a bit.